src/components/Contracts/Contracts.tsx

```diff
@@ -0,0 +1,198 @@
+import { ReactComponent as OpenIcon } from "../../img/icons/openIcon.svg"
+import { ReactComponent as CopyIcon } from "../../img/icons/copyIcon.svg"
+import { ReactComponent as UsdtIcon } from "../../img/icons/usdt.svg"
+import { ReactComponent as WbtcIcon } from "../../img/icons/wbtc.svg"
+import { ReactComponent as EthIcon } from "../../img/icons/eth.svg"
+
+const Contracts = () => {
```

Comment:
Please move this (and Deposit, Swap, Withdraw, Payments) to Components/FundYield/. And rename Contracts  to YieldInfo.